api/api.go

```diff
@@ -16,53 +16,39 @@ import (
 const (
 	defaultAPIVersion = "v1"
 
-	// StatusHealthy is the healthy status. This is determined based on status
+	// StatusSuccessful is the successful status. This is determined based on status
 	// type.
 	//
 	// Task Status: Determined by the success of a task updating. The 5 most
-	// recent task updates are stored as an ‘event’ in CTS. A task is healthy
-	// when all the stored events are successful.
-	//
-	// Overall Status: Determined by the health across all task statuses.
-	// Overall status is healthy when all task statuses are healthy.
```

Comment:
I removed Overall Status from these comments since overall status is returning a count of task-status rather than it's own interpretation on the health status values